src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -1957,6 +1962,29 @@ otError DBusThreadObjectRcp::SetNat64Cidr(DBusMessageIter &aIter)
 }
 #endif // OTBR_ENABLE_NAT64
 
+otError DBusThreadObjectRcp::GetEphemeralKeyEnabled(DBusMessageIter &aIter)
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(DBusMessageEncodeToVariant(&aIter, mBorderAgent.GetEpskcEnabled()) == OTBR_ERROR_NONE,
+                 error = OT_ERROR_INVALID_ARGS);
+
+exit:
+    return error;
+}
+
+otError DBusThreadObjectRcp::SetEphemeralKeyEnabled(DBusMessageIter &aIter)
+{
+    otError error = OT_ERROR_NONE;
+    bool    enable;
+
+    VerifyOrExit(DBusMessageExtractFromVariant(&aIter, enable) == OTBR_ERROR_NONE, error = OT_ERROR_INVALID_ARGS);
```

Comment:
`SuccessOrExit` should be more convenient in this use case.

Reply:
Accept. Thanks~